azure/apis/compute/v1alpha2/types.go

```diff
@@ -25,84 +25,104 @@ import (
 )
 
 const (
-	// ClusterProvisioningStateSucceeded is the state for a cluster that has succeeded provisioning
+	// ClusterProvisioningStateSucceeded is the state for a cluster that has
+	// succeeded provisioning.
 	ClusterProvisioningStateSucceeded = "Succeeded"
-	// DefaultReclaimPolicy is the default reclaim policy to use
+
+	// DefaultReclaimPolicy is the default reclaim policy to use.
 	DefaultReclaimPolicy = runtimev1alpha1.ReclaimRetain
-	// DefaultNodeCount is the default node count for a cluster
+
+	// DefaultNodeCount is the default node count for a cluster.
 	DefaultNodeCount = 1
 )
 
-// AKSClusterParameters define the configuration for AKS cluster resources
+// AKSClusterParameters define the desired state of an Azure Kubernetes Engine
+// cluster.
 type AKSClusterParameters struct {
-	// ResourceGroupName is the name of the resource group that the cluster will be created in
-	ResourceGroupName string `json:"resourceGroupName"` //--resource-group
+	// ResourceGroupName is the name of the resource group that the cluster will
+	// be created in
```

Comment:
I notice a number of these comments do not end with a period. Is that intentional?

Reply:
I'd prefer them to end with periods, but I'm sure I missed them in a bunch of places since I'm trying to power through all this pretty quickly.

Reply:
Sounds good! I'll try to go through and catch some, probably after v0.3 :)